src/pages/[lang]/informativa-privacy.tsx

```diff
@@ -33,19 +38,25 @@ declare const OneTrust: {
 };
 
 const PrivacyPage: NextPage = () => {
+  const { t } = useTranslation(["common", "privacy"]);
+
   useEffect(() => {
-    if (ONE_TRUST_PP_PAGE) {
-      OneTrust.NoticeApi.Initialized.then(() => {
-        OneTrust.NoticeApi.LoadNotices([ONE_TRUST_PP_PAGE], false);
-      });
-    }
+    const interval = setInterval(() => {
+      if (typeof OneTrust !== "undefined" && ONE_TRUST_PP_PAGE) {
+        clearInterval(interval);
+        OneTrust.NoticeApi.Initialized.then(() => {
+          OneTrust.NoticeApi.LoadNotices([ONE_TRUST_PP_PAGE], false);
+        });
+      }
+    }, 100);
+    return () => clearInterval(interval);
   }, []);
 
   return (
     <>
       <PageHead
-        title="SEND - Servizio Notifiche Digitali | Informativa Privacy"
-        description="Consulta l'informativa sul trattamento dei dati personali di SEND - Servizio Notifiche digitali"
+        title={t("title", { ns: "privacy" })}
+        description={t("description", { ns: "privacy" })}
       />
       <Script
```

Comment:
Try to use the onReady props on scripts instead of using interval. Do this on all pages that use OneTrust

Reply:
ok, I'll use the onReady prop also for 'note-legali' and and 'punti-di-ritiro/come-funziona'